xresources-theme.el

```diff
@@ -165,7 +165,7 @@
 
    ;; company-mode
    `(company-tooltip ((t (:foreground ,foreground :background ,background))))
-   `(company-tooltip-selection ((t (:foreground ,foreground :background ,background))))
+   `(company-tooltip-selection ((t (:foreground ,black :background ,white))))
```

Comment:
How about flipping the `,foreground` and `,background` colors, would the tooltip-selection still be visible enough?